### PR TITLE
Update the names of some icons and add a new category for token icons

### PR DESCRIPTION
**Pull request by cong1223**
Fix the correct name for the Cronos icon, update the name for the token, and add a new token icons category.

### CODE REVIEW

**Comment by cong1223** [comment]
In addition, the Dumi documentation content has also been updated.

File: .changeset/lazy-dolls-bake.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@ant-design/web3-icons': patch
+---
+
+Fix the correct name for the Cronos icon, update the name for the token, and add a new token icons category.
```
Comment (kiner-tang) [inline .changeset/lazy-dolls-bake.md:5]:
```suggestion
fix: Fix the correct name for the Cronos icon, update the name for the token, and add a new token icons category.
```

File: .dumi/theme/builtins/IconSearch/fields.ts [hunk 2]
```diff
@@ -13,26 +13,35 @@ const chain = [
...
 const tool = [
   'CoinbaseWallet',
   'Etherscan',
   'MetaMask',
   'WalletConnect',
   'TokenPocket',
   'Safeheron',
+  'Uniswap',
+  'PancakeSwap',
+  'SushiSwap',
```
Comment (yutingzhao1991) [inline .dumi/theme/builtins/IconSearch/fields.ts:35]: image

Due to duplicate IDs, when multiple icons are present on the page, there may be display issues.

You can fix it like: https://github.com/ant-design/ant-design-web3/pull/346

File: .changeset/lazy-dolls-bake.md (hunk 1 above)
Comment (cong1223) [reply .changeset/lazy-dolls-bake.md]: Thank you for the suggestion. The wording has been adjusted accordingly.

File: .dumi/theme/builtins/IconSearch/fields.ts (hunk 2 above)
Comment (cong1223) [reply .dumi/theme/builtins/IconSearch/fields.ts:35]: Thank you for the suggestion. The issue has been fixed.
image

**Review by yutingzhao1991 — APPROVED** [review]

**Pull request merged**